Using system portaudio and disabling portmixer seems to remove some -I...
src/AudioIOBase.cpp:80:47: error: member access into incomplete type 'const PaDeviceInfo'
src/AudioIOBase.h:25:8: note: forward declaration of 'PaDeviceInfo'
src/AudioIOBase.cpp:358:19: error: too many arguments to function call, expected 0, have 1; did you mean 'AudioIoCallback::IsStreamActive'?
<...>

Index: src/AudioIOBase.h
--- src/AudioIOBase.h.orig
+++ src/AudioIOBase.h
@@ -22,6 +22,8 @@ Paul Licameli split from AudioIO.h
 #include <wx/string.h>
 #include <wx/weakref.h> // member variable
 
+#include "../lib-src/portaudio-v19/include/portaudio.h"
+
 struct PaDeviceInfo;
 typedef void PaStream;
 
